test/query.test.js

```diff
@@ -419,9 +419,10 @@ describe('query', () => {
   test('take', () => {
     var p1 = assertQuery(query.Take(1, [1, 2]), [1])
     var p2 = assertQuery(query.Take(3, [1, 2]), [1, 2])
-    var p3 = assertQuery(query.Take(-1, [1, 2]), [])
+    // var p3 = assertQuery(query.Take(-1, [1, 2]), [])
```

Comment:
@marrony Have we made changes to the Take function? This third test no longer passes, however, this seems to be correct based on how Take currently works in the webshell.

Reply:
`Take` will accept negative numbers, this means it will take from the end of the array